modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java

```diff
@@ -401,11 +401,17 @@ private void _expandWildcards(Map<String, Object> typeSettings) {
 		for (Map.Entry<String, Object> entry : typeSettings.entrySet()) {
 			Object currentValue = entry.getValue();
 
-			if ((currentValue instanceof String) &&
-				_isWildcardValue((String)currentValue)) {
+			if (currentValue instanceof String) {
+				String currentValueString = (String)currentValue;
 
-				entry.setValue(
-					_getMatchingPaths(staticDirPath, (String)currentValue));
+				String key = StringUtil.toLowerCase(entry.getKey());
+
+				if (key.contains("url") &&
```

Comment:
@thiago-buarqque do we not also want to check whether the key contains "url" down below in the block starting with `currentValue instanceof List`? Or is that not a valid use-case?

Reply:
Not a valid use case, as of today we don't have a list of urls

Reply:
Ops, we do. Missed it, my bad

Reply:
@drewbrokke fixed it, sorry about that